instrumentation/resources/library/build.gradle.kts

```diff
@@ -84,6 +84,17 @@ testing {
 }
 
 tasks {
+  test {
+    dependsOn(jar)
+    doFirst {
+      // use test the final jar instead of directories with built classes to test the mrjar functionality
```

Comment:
```suggestion
      // used to test the final jar instead of directories with built classes to test the mrjar functionality
```

Reply:
I changed it to `use the final jar instead of directories with built classes to test the mrjar functionality` wdyt?

Reply:
lgtm